common/src/main/java/org/apache/gravitino/dto/authorization/SecurableObjectDTO.java

```diff
@@ -31,44 +31,41 @@
 /** Data transfer object representing a securable object. */
 public class SecurableObjectDTO implements SecurableObject {
 
-  @JsonProperty("fullName")
```

Comment:
why changing this class?

Reply:
When I added the e2e tests, I find an issue. So I fix it by the way.

Reply:
The PR has mixed fixing, could you add more contexts in `What changes were proposed in this pull request` ?

Reply:
Done.